test/runner_test.exs

```diff
@@ -21,11 +21,6 @@ defmodule RunnerTest do
   end
 
   test "user can choose default markers" do
-    response = fn ->
-      assert Runner.use_default_markers?("Y") == :ok
-    end
-
-    assert capture_io(response) ==
-             "Great!\n\nPlayer One - Your marker is \"X\".\nPlayer Two - Your marker is \"O\".\n\n"
+    assert Runner.use_default_markers?("Y") == {"X", "O"}
```

Comment:
Curious as to why you deleted the capture_io here? Getting some test output now. If you just want to test the return value and not the message you can do:
```elixir
test "user can choose default markers" do
  response = fn ->
    assert Runner.use_default_markers?("Y") == {"X", "O"}
  end
  capture_io(response)
end
```

Reply:
Yes, not sure why I didn't just leave the capture_io there...